src/hl/ready.rs

```diff
@@ -195,20 +287,11 @@ where
 
         match send_time {
             SendTime::Delayed(time) => {
-                // Panic if the time is not rounded to top 31 bits
```

Comment:
This is actually intentional. Delayed TX with a timestamp not rounded should be a hard error, this leads to very hard to diagnose inaccuracies.